src/components/Nfts/NftTable.tsx

```diff
@@ -38,7 +36,7 @@ export const NftTable = () => {
 
   const [networkFilters, setNetworkFilters] = useState<ChainId[]>([])
 
-  const isLoading = useAppSelector(selectGetNftUserTokensPending)
+  const { isLoading } = useFetchNfts()
```

Comment:
Perhaps `useNFTs` is a better terminology for this hook as it doesn't re-fetch, fetching is just part of this does (or we could use the `query` terminology which clearly describes the intent, at the end of the day this is similar to a query)